src/core/thread/dua_manager.cpp

```diff
@@ -307,6 +307,8 @@ void DuaManager::HandleNotifierEvents(Events aEvents)
 {
     Mle::MleRouter &mle = Get<Mle::MleRouter>();
 
+    VerifyOrExit(mle.IsAttached());
```

Comment:
Should we stop the reregistration/registration timer when the node is detached?

Reply:
In `void DuaManager::HandleTimeTick(void)`, the registration is updated per ticker
https://github.com/openthread/openthread/blob/8dc9a53cc7530495023e10949cbafb4c4d18ae76/src/core/thread/dua_manager.cpp#L396
Updated to `VerifyOrExit(mle.IsAttached(), mDelay.mValue = 0);` to reset the delay value when detached, and ensure it is stopped.
Pls take another review.